os/android/gralloc1bufferhandler.cpp

```diff
@@ -153,8 +153,6 @@ bool Gralloc1BufferHandler::CreateBuffer(uint32_t w, uint32_t h, int format,
     uint64_t modifier = 0;
```

Comment:
better initialize as: uint64_t modifier = DRM_FORMAT_MOD_NONE;

Reply:
this may require to include drm header, still need to consider it

Reply:
@YuanjunHuang Done

Reply:
LGTM